doc/classes/Camera3D.xml

```diff
@@ -202,12 +209,24 @@
 		<member name="near" type="float" setter="set_near" getter="get_near" default="0.05">
 			The distance to the near culling boundary for this camera relative to its local Z axis. Lower values allow the camera to see objects more up close to its origin, at the cost of lower precision across the [i]entire[/i] range. Values lower than the default can lead to increased Z-fighting.
 		</member>
+		<member name="oblique_normal" type="Vector3" setter="set_oblique_normal" getter="get_oblique_normal" default="Vector3(0, 1, 0)">
+			The desired normal vector of the world space plane used by an oblique camera as an oblique near clipping plane.
```

Comment:
Should the normal vector of the mentioned world space plane be provided in World space or in View space to this function ? Either way this should be clarified.

Reply:
Gave it further thinking, and in the end I believe the oblique plane's normal and position attributes should be expressed in view space for consistency.
In other words these shouldn't vary when the camera moves around (we're setting camera attributes after all, not scene attributes).
You purposefully provide 
`set_oblique_plane_from_transform()` that allows users to set the oblique plane's attributes in world space already.